src/main/java/edu/ucsb/cs156/gauchoride/controllers/IndividualRidesController.java

```diff
@@ -0,0 +1,158 @@
+package edu.ucsb.cs156.gauchoride.controllers;
+
+import edu.ucsb.cs156.gauchoride.entities.IndividualRides;
+import edu.ucsb.cs156.gauchoride.errors.EntityNotFoundException;
+import edu.ucsb.cs156.gauchoride.repositories.IndividualRidesRepository;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import lombok.extern.slf4j.Slf4j;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+@Api(description = "IndividualRides")
+@RequestMapping("/api/individualRides")
+@RestController
+@Slf4j
+public class IndividualRidesController extends ApiController {
+ 
+    @Autowired
+    IndividualRidesRepository individualRidesRepository;
+    
+    @ApiOperation(value = "List all individualRides")
+    @PreAuthorize("hasRole('ROLE_USER')")
```

Comment:
```suggestion
    @PreAuthorize("hasRole('ROLE_DRIVER') || hasRole('ROLE_ADMIN') || hasRole('ROLE_RIDER')")
```

ROLE_USER should not have access any sensitive information, since anyone/everyone with a Google Account has access to ROLE_USER just by logging in.